src/config.js

```diff
@@ -139,7 +139,13 @@ global.config = {
       1800: [8, 15], // RCL 5
       2300: [11, 21], // RCL 6
     },
+    transferToCarry: false,
```

Comment:
Not sure what the intention is.

The default behavior should be that carries transfer to carries. If I understand the implementation correct, that would be disabled here.
And there are two methods, something like `transferToCreep` which actually transferrs and something like `checkForTransfer` which lets the carry move in the other direction because it will get energy in this tick.

Both needs to be adapted.

Reply:
Maybe we can swap carries' target when they transfer to each other.